scribble/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+
+  def show
+    @comment = Comment.find(params[:id])
+  end
+
+  def new
+    @comment = comment.new
```

Comment:
think you need to instance variables defined - missing one for the `Post` this new comment should be associated to